Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'solidus', github: 'solidusio/solidus', branch: 'v1.1'
```

Comment:
@tvdeyen that's correct, I think only fetching from branch 'v1.1' should be updated with: 
```
gem 'solidus', '~> 1.1'
```
If it's ok for you, I'll update my PR :wine_glass: